src/pykiso/tool/pykiso_to_pytest/cli.py

```diff
@@ -0,0 +1,174 @@
+##########################################################################
+# Copyright (c) 2010-2022 Robert Bosch GmbH
+# This program and the accompanying materials are made available under the
+# terms of the Eclipse Public License 2.0 which is available at
+# http://www.eclipse.org/legal/epl-2.0.
+#
+# SPDX-License-Identifier: EPL-2.0
+##########################################################################
+
+import pathlib
+import re
+from pathlib import Path
+from typing import Any, List, Optional, Tuple, Union
+
+import click
+import isort
+from black import FileMode, format_str
+from jinja2 import Environment, FileSystemLoader
+
+from pykiso import config_parser
+
+ROOT_PATH = pathlib.Path(__file__).parent.resolve()
+
+
+def format_value(value: Any) -> Union[Any, str]:
+    """Add double quotes to a string if value is
+    of type string. Else return original value
+
+    :param value: value to evaluate
+    :return: quoted string if value was a string
+    """
+
+    if isinstance(value, (str)):
+        return f'"{value}"'
+
+    return value
+
+
+def find_string_in(
+    nested_dict: dict, search_value: str, prepath: Tuple[str, ...] = ()
+) -> Optional[Tuple[str, ...]]:
+    """find if string is part of an value in a nested dictionary.
+
+    :param nested_dict: dictionary to search in
+    :param search_value: string to search for
+    :param prepath: path storage for recursive call, defaults to ()
+    :return: path to locataion when found else None
+    """
+    for key, value in nested_dict.items():
+        path = prepath + (key,)
+        if isinstance(value, str) and search_value in value:  # found value
+            return path
+        elif isinstance(value, dict):
+            key_path = find_string_in(value, search_value, path)
+            if key_path is not None:
+                return key_path
+    # If not found
+    return None
+
+
+def nested_get(dic: dict, keys: Tuple[str, ...], copy: bool) -> Any:
+    """Get dictionary items from list of keys
+
+    :param dic: dictionary to search in
+    :param keys: key path as list
+    :return: element at located position
+    """
+    for key in keys:
+        dic = dic[key]
+
+    if copy:
+        return dict(dic)
+
+    return dic
+
+
+def remove_key(dic: dict, key_path: Tuple[str, ...]) -> None:
+    """Remove dictionary element at given location.
+
+    :param dic: dictionary where the key shall be removed
+    :param key_path: key path as list
```

Comment:
Tuple or list. This is actually important, because one is mutable, the other not.

Reply:
refactored to list